### DISCUSSION TITLE
Webhook response [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5859] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### can you help us identify anomaly observed between responses?  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### we are getting reponses like this how to handle this?  
![image0](https://cdn.discordapp.com/attachments/1101497389522620566/1102506127507341332/image.png)  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### I have inserted same data in the table and it has given us different type of response in body so how can we handle this?  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### Scenario:   
####    
#### I have 2 main fields in our table: 1 is a json field and another one is a relation field. I am inserting the record through api like first creating a record with json data (https://all-apis.nocodb.com/#tag/DB-Table-Row/operation/db-table-row-create) and then connecting relation field data (https://all-apis.nocodb.com/#tag/DB-Table-Row/operation/db-table-row-nested-add). I have created a webhook on this table for inserting record as i need both the fields in the payload.  
![image0](https://cdn.discordapp.com/attachments/1101497389522620566/1102508431824392262/image.png)  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### Any solution to this?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1101497389522620566)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
